src/unity/lib/variant.cpp

```diff
@@ -10,6 +10,8 @@
 #include <unity/lib/api/unity_sarray_interface.hpp>
 #include <unity/lib/api/unity_graph_interface.hpp>
 #include <unity/lib/api/function_closure_info.hpp>
+#include <unity/lib/api/plot_interface.hpp>
```

Comment:
Making `Plot` part of `variant_type` seems like overkill. I thought we planned to use the toolkit SDK to expose the Plot object - what changed? Does the old-style Cython interface not allow returning types exposed through the toolkit SDK?